# Goal Position
uint8[] id
float32[] goal_position

